commercetools-models/src/main/java/io/sphere/sdk/shippingmethods/commands/updateactions/RemoveZone.java

```diff
@@ -14,18 +15,18 @@
  * {@include.example io.sphere.sdk.shippingmethods.commands.ShippingMethodUpdateCommandIntegrationTest#workingWithZones()}
  */
 public final class RemoveZone extends UpdateActionImpl<ShippingMethod> {
-    private final Reference<Zone> zone;
+    private final ResourceIdentifier<Zone> zone;
 
-    private RemoveZone(final Reference<Zone> zone) {
+    private RemoveZone(final ResourceIdentifier<Zone> zone) {
         super("removeZone");
         this.zone = zone;
     }
 
-    public Reference<Zone> getZone() {
+    public ResourceIdentifier<Zone> getZone() {
```

Comment:
I would suggest to mention this change in the release notes too. May be like telling our users that we changed all update actions which referenced a zone to use a `ResourceIdentifier<Zone>` instead of a `Reference<Zone>`. What do yo think about it?

Reply:
totally agree, would add it.